### DISCUSSION TITLE
V2 Migration

### DISCUSSION THREAD

**Discussion by hrohwer** [automadcms/discussions#21] [General]
Hi, FYI, I have tried the migration process on a simple site and the pages folders end up empty and I have no landing page.
Kind regards

**Comment by marcantondahmen** [comment]
Hi, it is difficult to help without knowing what you did exactly, what versions did you use and so on ...

**Reply by hrohwer** [reply]
I understand, I should have taken note of that, I abandoned the migration and started fresh to play with V2. Thank you

**Reply by hrohwer** [reply]
On Another note .... the pretty urls don't work on a litespeed server at the moment.

**Reply by hrohwer** [reply]
self::set('AM_BASE_INDEX', AM_BASE_URL);    I removed this to get it to work . AM_INDEX);

**Reply by marcantondahmen** [reply]
Only Apache and Nginx are supported out of the box as mentioned in the system requirements. In order to use LiteSpeed you have to set `AM_INDEX` in `config/config.php` to an empty string. And don't change core files. Changes there are not persistent.
